src/modules/commentModal.js

```diff
@@ -55,31 +55,32 @@ const commentModal = async (show) => {
     // fetching involvment API
     let commentsContainer;
     let commentsHeading;
-
-    const response = await TVShowApp.getComments(show.id)
-    console.log(response);
-    if (response.length > 0){
-      commentsContainer = document.createElement('div');
-      commentsContainer.className = 'comments-container';
-      response.forEach((data)=>{
-          const comment = document.createElement('p');
-          comment.className = 'comment';
-          comment.textContent = `${data.creation_date} ${data.username} ${data.comment}`
-          commentsContainer.appendChild(comment);
-
-      })
-      commentsHeading = document.createElement('h5');
-      commentsHeading.className = 'comments-heading';
-      commentsHeading.textContent = `Comments (${response.length})`;
-    }
-    else{
-      commentsContainer = document.createElement('div');
-      commentsContainer.className = 'comments-container';
-      commentsContainer.textContent = 'No comments found';
-      commentsHeading = document.createElement('h5');
-      commentsHeading.className = 'comments-heading';
-      commentsHeading.textContent = 'Comments (0)';
```

Comment:
Great job on implementing the save time in real time functionality, I would suggest to not create html elements in javaScript files, this could increase the size of files and could decrease chances of of debugging. Kindly for clear code and javaScript best practices change it.